lib/SILOptimizer/Transforms/SILSROA.cpp

```diff
@@ -286,9 +287,27 @@ void SROAMemoryUseAnalyzer::chopUpAlloca(std::vector<AllocStackInst *> &Worklist
   eraseFromParentWithDebugInsts(AI);
 }
 
-static bool runSROAOnFunction(SILFunction &Fn) {
-  std::vector<AllocStackInst *> Worklist;
+bool swift::runSROAOnInsts(llvm::ArrayRef<AllocStackInst *> Worklist) {
```

Comment:
I think ArrayRef is already available here so no need for llvm:: namespace. Is that not the case?

Reply:
Nit: if it’s not an array to store temporaries in an iterative algorithm, it’d be more appropriate to call this operand “insts” or something. The small vector created in the function body is the actual work list.

Reply:
Agreed, you can drop the llvm:: here, and also in the SmallVector declarations on line 292/310

Reply:
Done.